From ccc63d08a0ec62bff84b412e0a22fe7338dac718 Mon Sep 17 00:00:00 2001
From: Bart Oostveen <bart@bartoostveen.nl>
Date: Wed, 21 Oct 2026 01:43:52 +0200
Subject: [PATCH] nextcloud{33,34,35}: use symlink to cacerts instead of
 vendored bundle

Opted to use `ca-no-trust-rules-bundle.crt` as there are do not seem to be trust rules in Nextcloud (most PHP applications don't anyway)

Related to: https://github.com/NixOS/nixpkgs/issues/576541
---
 pkgs/servers/nextcloud/default.nix | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/pkgs/servers/nextcloud/default.nix b/pkgs/servers/nextcloud/default.nix
index f2e9ce3917c0..780b11cf6081 100644
--- a/pkgs/servers/nextcloud/default.nix
+++ b/pkgs/servers/nextcloud/default.nix
@@ -2,6 +2,7 @@
   lib,
   stdenvNoCC,
   fetchurl,
+  cacert,
   nixosTests,
   nextcloud33Packages,
   nextcloud34Packages,
@@ -31,8 +32,13 @@ let
 
       installPhase = ''
         runHook preInstall
+
         mkdir -p $out/
         cp -R . $out/
+
+        rm $out/resources/config/ca-bundle.crt
+        ln -s ${cacert}/etc/ssl/certs/ca-no-trust-rules-bundle.crt $out/resources/config/ca-bundle.crt
+
         runHook postInstall
       '';
 
-- 
2.55.0

